packages/inferno-router/README.md

```diff
@@ -21,9 +21,11 @@ Features added from react-router@5:
 Features added from react-router@6:
 - Async data fetching before navigation using [`loader`-attribute](https://reactrouter.com/en/main/route/loader). See [demo](https://github.com/infernojs/inferno/tree/master/demo/inferno-router-demo).
 
-NOTE: While we want the basic fetch behaviour is the same as react-router@6, we are currently missing:
-- progress bar support
+NOTE: While we want the basic fetch behaviour to be the same as react-router@6, we are currently missing:
+- download progress support
 - form submission
+- redirect support
+- not exposing headers, type or status code to render method
```

Comment:
Hmm, why should we follow react router implementation?

I would change wording to just mention following features are not supported yet. 

But is it possible for application side to provide its own settings to fetch, thus implementing those are possible at application level?

Reply:
> Hmm, why should we follow react router implementation?
My thoughts were that it is a react-router port from the outset so we should try to keep it compatible where we can.

> 
> I would change wording to just mention following features are not supported yet.
WILL FIX

> 
> But is it possible for application side to provide its own settings to fetch, thus implementing those are possible at application level?
Yes, the application is responsible for implementing the fetch call (or whatever library they want to use) so they have full control. Note: adding support for exposing response headers, form submission and redirect support will be a non-breaking change from what I can tell.